src/explore-education-statistics-frontend/src/middleware/pages/__tests__/redirectPages.test.ts

```diff
@@ -11,317 +11,457 @@ const redirectService = _redirectService as jest.Mocked<
 >;
 
 describe('redirectPages', () => {
-  const redirectSpy = jest.spyOn(NextResponse, 'redirect');
-  const nextSpy = jest.spyOn(NextResponse, 'next');
+  interface TestDataWithoutRedirect {
+    routePattern: string;
+    fromSlug: string;
+  }
+
+  interface TestDataWithRedirect extends TestDataWithoutRedirect {
+    toSlug: string;
+  }
+
+  const slugPlaceholder = '{slug}';
 
   const testRedirects: Redirects = {
-    methodologies: [
-      { fromSlug: 'original-slug-1', toSlug: 'updated-slug-1' },
-      { fromSlug: 'original-slug-2', toSlug: 'updated-slug-2' },
-    ],
-    publications: [
-      { fromSlug: 'original-slug-3', toSlug: 'updated-slug-3' },
-      { fromSlug: 'original-slug-4', toSlug: 'updated-slug-4' },
-    ],
+    methodologies: [{ fromSlug: 'original-slug-1', toSlug: 'updated-slug-1' }],
+    publications: [{ fromSlug: 'original-slug-2', toSlug: 'updated-slug-2' }],
+    releases: [{ fromSlug: 'original-slug-3', toSlug: 'updated-slug-3' }],
```

Comment:
I was expecting something different for the format of `RedirectsViewModel`. 

I was expecting each release directly to be directly tied to a specific publication. If they're not tied together, then if you have a redirect for release slug `2000`, I think that'll effect all publication's with a release `2000`? So analysts could introduce a new release slug, but then unintentionally change the release slug for a different publication?

Reply:
We have decided to do the BE changes as part of another PR for [EES-5632](https://dfedigital.atlassian.net/jira/software/c/projects/EES/boards/105?selectedIssue=EES-5632).

This PR will be blocked for merging until that work is done. I have updated the description of this PR to reflect this.